pom.xml

```diff
@@ -19,7 +19,7 @@
         <solr.version>8.11.1</solr.version>
         <logback.version>1.2.11</logback.version>
         <jackson.version>2.13.2</jackson.version>
-        <hibernate.version>5.4.33.Final</hibernate.version>
+        <hibernate.version>5.6.11.Final</hibernate.version>
         <spring-security.version>5.6.4</spring-security.version>
```

Comment:
Probably should upgrade spring libs to latest. Right now Spring is 5.3.23 and security is 5.7.4